ADALiOS/ADALiOS/ADUserInformation.h

```diff
@@ -38,6 +38,8 @@
 /*! Determines whether userId is displayable */
 @property (readonly) BOOL userIdDisplayable;
 
+@property (readonly) NSString* uniqueId;
```

Comment:
I didn't want to diverge too much from established ADALiOS behavior, at least in the default case. That means having RequiredDisplayableId mean the previous behavior. This is something we can fix in the redesign.